net-mgmt/lldpd/src/opnsense/service/conf/actions.d/actions_lldpd.conf

```diff
@@ -0,0 +1,35 @@
+[start]
+command:/usr/local/opnsense/scripts/OPNsense/Lldpd/setup.sh;/usr/local/etc/rc.d/lldpd start
+parameters:
+type:script
+message:starting Lldpd
+
+[stop]
+command:/usr/local/etc/rc.d/lldpd stop; exit 0
```

Comment:
does this really require masking the exit code?

Reply:
I'll try it :)